lib/widgets/classify_species.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:convert';
+
+import 'package:http/http.dart';
+import 'package:flutter/services.dart' show rootBundle;
+
+
+class SpeciesClassifier {
+  var queryParameters = {
+    'topK': '1',
+    'predictMode': 'classifyAndDetect',
+  };
+  Map<String, String> headers = {
+    'Content-Type': 'application/octet-stream',
+    'Ocp-Apim-Subscription-Key': '485afbef61f9444bbcb4b454ef35d966'
+  };
+
+  _makePostRequest(var imagePath) async {  // set up POST request arguments
+    //var f = File(imagePath);
+    var bytes = await rootBundle.load("assets/image.jpg");
```

Comment:
Can't we just use the `imagePath` parameter here since we're already passing assets/image.jpg for testing as the value?